features/adb-openshift.feature

```diff
@@ -51,31 +47,10 @@ Feature: Command output from various OpenShift related commands in ADB
     DOCKER_REGISTRY=hub.openshift.centos7-adb.<ip>.xip.io
     """
 
-    When I successfully run `bundle exec vagrant service-manager env`
-    Then stdout from "bundle exec vagrant service-manager env" should contain "export DOCKER_HOST=tcp://<ip>:2376"
-    And stdout from "bundle exec vagrant service-manager env" should match /export DOCKER_CERT_PATH=.*\/.vagrant\/machines\/cdk\/virtualbox\/docker/
-    And stdout from "bundle exec vagrant service-manager env" should contain "export DOCKER_TLS_VERIFY=1"
-    And stdout from "bundle exec vagrant service-manager env" should match /export DOCKER_API_VERSION=1.2\d/
-    And stdout from "bundle exec vagrant service-manager env" should match /# eval "\$\(vagrant service-manager env\)"/
-    And stdout from "bundle exec vagrant service-manager env" should contain:
-    """
-    # openshift env:
-    # You can access the OpenShift console on: https://<ip>:8443/console
-    # To use OpenShift CLI, run: oc login https://<ip>:8443
-    export OPENSHIFT_URL=https://<ip>:8443
-    export OPENSHIFT_WEB_CONSOLE=https://<ip>:8443/console
-    export DOCKER_REGISTRY=hub.openshift.centos7-adb.<ip>.xip.io
-
-    # run following command to configure your shell:
-    # eval "$(vagrant service-manager env)"
-    """
-
-    When I successfully run `bundle exec vagrant up --provider <provider>`
-    And I run `bundle exec vagrant service-manager install-cli openshift`
+    When I run `bundle exec vagrant service-manager install-cli openshift`
```

Comment:
@LalatenduMohanty the command `bundle exec vagrant service-manager env` is covered in `env-command.feature` file.
This env command for openshift has been covered in test case above for this deleted lines.